keycloak-provisioning/terraform.tf

```diff
@@ -5,20 +5,13 @@ variable "default_password" {
 }
 
```

Comment:
Update this ref because you're using 4.4:
from
```
https://registry.terraform.io/providers/mrparkers/keycloak/4.1.0/docs/resources/openid_client
```
to
```
https://registry.terraform.io/providers/mrparkers/keycloak/4.4.0/docs/resources/openid_client
```